zavod/docs/usage.md

```diff
@@ -35,4 +35,9 @@ $ zavod crawl ...
 Then run the exporter with `--clear` to ensure the latest statements are included in the output:
 
 ```bash
-$ zavod export --clear ...```
\ No newline at end of file
+$ zavod export --clear ...
+```
+
+# Debugging Crawlers in VSCode
+It is possible to debug crawlers through the Python debugger that comes with the standard VSCode install. To enable it either rename `.vscode/launch.json.example` to `launch.json` or copy over the launch configuration you find in it to your own `launch.json` file. 
```

Comment:
As a vscode noob, this was the only thing I didn't immediately realise before awesomeness.

```suggestion
It is possible to debug crawlers through the Python debugger that comes with the standard VSCode install. To enable it either rename `.vscode/launch.json.example` to `.vscode/launch.json` or copy over the launch configuration you find in it to your own `launch.json` file. 
```